src/ir_Bosch.cpp

```diff
@@ -273,7 +273,7 @@ String IRBosch144AC::toString(void) const {
                            static_cast<int>(stdAc::fanspeed_t::kAuto),
                            static_cast<int>(stdAc::fanspeed_t::kAuto),
                            static_cast<int>(stdAc::fanspeed_t::kMedium));
-  result += addTempToString(getTemp());
```

Comment:
Ditto comment above.

Reply:
Should I use `addTempFloatToString` function?

Reply:
Good catch. It should now be `addTempFloatToString`